api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/FindPatientQueryBuilder.java

```diff
@@ -0,0 +1,15 @@
+package org.openmrs.module.registrationcore.api.mpi.openempi;
+
+import org.openmrs.Patient;
+
+public class FindPatientQueryBuilder {
```

Comment:
just glancing at this class, I feel like the code would be clearer if this and other methods were pulled together into a class like OpenEmpiConnector